src/Humanizer/Bytes/ByteRate.cs

```diff
@@ -76,5 +76,58 @@ public string Humanize(string format, TimeUnit timeUnit = TimeUnit.Second)
             return new ByteSize(Size.Bytes / Interval.TotalSeconds * displayInterval.TotalSeconds)
                 .Humanize(format) + '/' + displayUnit;
         }
+        /// <summary>
+        /// Returns the humanized string with default parameters.
+        /// </summary>
+        /// <returns></returns>
+        public override string ToString()
+        {
+            return Humanize();
+        }
+
+        /// <summary>
+        /// Returns a humanized string of the current rate object using the supplied parameters
+        /// </summary>
+        /// <param name="timeUnit">Unit of time to calculate rate for (defaults is per second)</param>
+        /// <param name="format">The string format to use for the number of bytes</param>
+        /// <returns></returns>
+        public string ToString(string format, TimeUnit timeUnit = TimeUnit.Second)
+        {
+            return Humanize(format, timeUnit);
+        }
+
+        /// <inheritdoc />
+        /// <summary>
+        /// Compares the current ByteRate object to another supplied ByteRate object.
+        /// Rates are normalized before comparing, e.g. 60Mb/Min is equal to 1024KB/sec
+        /// </summary>
+        /// <param name="other">The ByteRate object to use for the comparison</param>
+        /// <returns>0 if the rates are equivalent, -1 if lower than the 'other' object, 1 if higher than the 'other' object</returns>
+        public int CompareTo(ByteRate other)
+        {
+            var left = Size.Bytes / Interval.TotalSeconds;
+            var right = other.Size.Bytes / other.Interval.TotalSeconds;
+            if (left < right) return -1;
+            return right < left ? 1 : 0;
+        }
+
+        /// <inheritdoc />
+        /// <summary>
+        /// Checks if two ByteRate objects have the same equivalent rate
+        /// </summary>
+        /// <param name="other"></param>
+        /// <returns>True if rates are equivalent, otherwise false</returns>
+        public bool Equals(ByteRate other)
+        {
+            return CompareTo(other) == 0;
```

Comment:
Is there a faster/more direct equality check than through the compare?

Reply:
Not that I can think of without either: duplicating code, or losing the sense of equivalence that the final rate is the important factor, not the given ByteSize and Interval values. But perhaps a small amount of code duplication is OK to keep the IEquatable<T> and IComparable<T> methods not dependent on each other...
```suggestion
            return (Size.Bytes / Interval.TotalSeconds) - (other.Size.Bytes / other.Interval.TotalSeconds) == 0;
```